website/public/css/global-colors.styl

```diff
@@ -19,6 +19,7 @@ $color-mp = $best
 $color-toolbar = lighten($color-herobox, 70%)
 $color-options-menu = lighten($color-herobox, 85%)
 $color-options-submenu = lighten($color-herobox, 75%)
+$color-options-blurb = lighten($color-herobox, 75%)
```

Comment:
I like having the specific color here, but let's make it: `$color-options-submenu-blurb = $color-options-submenu`